pom.xml

```diff
@@ -159,6 +169,7 @@ SOFTWARE
             <configuration>
               <excludes combine.children="append">
                 <exclude>pmd:.*</exclude>
+                <exclude>checkstyle:.*</exclude>
```

Comment:
@Graur this is dangerous. Why? Because of the bug in Qulice? If so, would be great to add a comment here, so that we know why Checkstyle is disabled.